src/interactions/abstractInteraction.ts

```diff
@@ -18,6 +18,10 @@ export module Interaction {
       this._hitBox = hitBox;
     }
 
+    public _requiresHitbox() {
```

Comment:
This endpoint will be removed eventually correct?  Can you place a ticket on eventually removing thehitbox idea and reference that issue here?